src/xdocs/checks/misc/finalparameters.xml

```diff
@@ -144,6 +152,57 @@ public class Example3 {
   public void methodOne(final int x) { }
   public void methodTwo(int x) { }
   public static void main(String[] args) { } // violation, 'args should be final'
+}
+        </source>
+        <p id="Example4-config">
+          To configure the check to enforce final parameters for catch and
+          for-each blocks while ignoring unnamed parameters.
+          Since unnamed parameters cannot be read from or written to,
+          marking them as final is unnecessary:
```

Comment:
The wording is a bit weird. We usually say "To configure...: config example" (notice no period so it is a complete thought), but instead there is another sentence with an explanation instead of a config, making the first sentence not really a complete sentence since it says "To configure" without showing them how to configure.

I would remove the second sentence, replace the period of the first sentence with `:`, and use the Java code comments as a way to convey the reason you tried to show in the second sentence. If you really needed to explain more, it should be done before "To configure" or after the Java example IMO.

Reply:
done completely removed it the code examples and code comments speak well for this point.